src/Client/app/(root)/my-exams/[id]/page.tsx

```
@@ -1,415 +1,474 @@
+                            <p>Time Left : </p>
+                            <p
+                                className={`font-mono ml-1 ${
+                                    timeLeft !== undefined && timeLeft < 300
+                                        ? "text-danger"
+                                        : "text-success"
+                                }`}
+                            >
+                                {timeLeft !== undefined
+                                    ? FormatTimeHourMinutesSeconds({
+                                          seconds: timeLeft,
+                                      }) + " s"
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)